components/SearchBar/index.tsx

```diff
@@ -28,27 +28,39 @@ function Component() {
   const Router = useRouter();
   const net = useStore($net);
   const zil_address = useStore($wallet);
-  const user = useStore($user); //@todo learn the difference between useStore & getState to share in the sprint3 file
+  const user = useStore($user);
+  const username = user?.name!;
+  const domain = user?.domain!;
+  const is_admin = useStore($isAdmin);
 
-  const [input, setInput] = useState("");
-  const [thisName, setName] = useState("");
-  const [thisDomain, setDomain] = useState("");
   const [loading, setLoading] = useState(false);
   const [error, setError] = useState("");
+  const [search, setSearch] = useState("");
+
+  const spinner = (
+    <i className="fa fa-lg fa-spin fa-circle-notch" aria-hidden="true"></i>
+  );
 
   // Resolve URL
 
   const checkPath = () => {
     const input = window.location.pathname.replace("/", "").toLowerCase();
     if (input === "") {
       return false;
+    } else if (
+      input === "buynftusername" ||
+      input === "didxwallet" ||
+      input === "treasury" ||
+      input === "verifiablecredentials" ||
+      input === "xpoints"
+    ) {
+      return false;
     } else if (
       input.split("/")[1] === "did" ||
-      input.split("/")[1] === "buy" ||
       input.split("/")[1] === "xwallet" ||
-      input.split("/")[1] === "funds" ||
       input.split("/")[1] === "recovery" ||
-      input.split(".")[1] === "tyron" ||
```

Comment:
.tyron shall redirect to GitHub since smart contracts are called, e.g. xwallet.tyron

Reply:
copy on this

Reply:
this means that ssibrowser.com/xwallet.tyron should act like typing xwallet.tyron on the search bar